### ISSUE TITLE
get_kline function isn't returning specified dates/times

### ISSUE THREAD

**Issue by 05remla** [Kucoin/kucoin-python-sdk#47]
import datetime
import time

t_start = datetime.datetime(2021, 1, 1)
t_start = time.mktime(t_start.timetuple())

t_stop = datetime.datetime(2021, 2, 1)
t_stop = time.mktime(t_start.timetuple())

data = marketclient.get_kline('REQ-BTC', '1min', startAt=t_start, endAt=t_stop)

all returned data is the present day - 1500 pieces of data, whether that be 1min or 1day etc...

**Comment by Tjorriemorrie** [comment]
Yeah I have the exact same problem, except it is 100 items being returned for me. Also the docs shows `:param kwargs: [Optional] startAt, endAt, currentPage, pageSize` where the paging does not exist.

**Comment by kevwilhelm95** [comment]
I am also experiencing this issue. If anyone has any information please update
